telegram/_telegramobject.py

```diff
@@ -81,6 +86,110 @@ def __getitem__(self, item: str) -> object:
                 f"`{item}`."
             ) from exc
 
+    def __eq__(self, other: object) -> bool:
```

Comment:
tbh I find this moving of methods rather irritating 😄 if you didn't change anything in them, I'd prefer it if they just stayed where they are. makes git blame more reliable & makes reviewing easier